rdflib/query.py

```diff
@@ -37,7 +37,9 @@ class Processor(object):
     def __init__(self, graph):
         pass
 
-    def query(self, strOrQuery, initBindings={}, initNs={}, DEBUG=False):
+    def query(
+        self, str_or_query, initBindings={}, initNs={}, DEBUG=False  # noqa: N803
```

Comment:
I still would prefer we don't break our API just for this, or at least split the interface breaking into another PR.

```suggestion
        self, strOrQuery, initBindings={}, initNs={}, DEBUG=False  # noqa: N803
```

Reply:
I don't have a position on this but my model is letting me down here. If `strOrQuery` isn't a kwarg, how does changing it break the API?

Reply:
someone can be calling `Graph.query(strOrQuery="SELECT * WHERE { ?s ?p ?o }")` as by default all arguments can be used as keywords or positional arguments, and if they are then changing `strOrQuery` to `str_or_query` will cause their code to fail.

Since python 3.8 there is an option for making arguments positional only [[PEP-0570](https://peps.python.org/pep-0570/)], so for example `def query(self, str_or_query, /, initBindings={}, initNS={}, DEBUG=false)` - and with a positional only arg it is impossible to call it using a keyword, but without this all args can be used as keywords.

Reply:
> by default all arguments can be used as keywords or positional arguments
> 
> Since python 3.8 there is an option for making arguments positional only ... and with a positional only arg it is impossible to call it using a keyword, but without this all args can be used as keywords.

Oh, I didn't know that. Thanks for the explanation. Will revert it.

Reply:
Reverted.